openfoodfacts/types.py

```diff
@@ -870,6 +870,7 @@ def check_user_agent(self):
 class DatasetType(str, enum.Enum):
     csv = "csv"
     jsonl = "jsonl"
+    obsolete = "obsolete"
```

Comment:
technically, its a `jsonl` type.. so maybe rename to `obsolete_jsonl` instead ?
I'll let @raphael0202 decide

Reply:
I would rather add an `obsolete` parameter to the function that having a new dataset type. It does not really make sense here to add `obsolete` as dataset type, as dataset type currently indicates the export format.

Reply:
i also think an additional parameter is more logical.
only issue is that it only currently applies to the off flavor

Reply:
I've updated the implementation to use an `obsolete` parameter instead of adding a new DatasetType. Please review the changes! @raphael0202

Thanks